sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Queues/QueueProcessor.cs

```diff
@@ -89,17 +89,17 @@ internal protected virtual async Task CompleteProcessingMessageAsync(QueueMessag
         {
             if (result.Succeeded)
             {
-                await DeleteMessageAsync(message, cancellationToken).ConfigureAwait(false);
+                await DeleteMessageAsync(message, CancellationToken.None).ConfigureAwait(false);
```

Comment:
I don't think we should be ignoring the cancellationTokens passed in - it should instead be up to the caller to decide to pass None in for operations that shouldn't be cancelled. I believe you're doing that in the QueueProcessor already, passing CancellationToken.None to CompleteProcessingMessageAsync.